torch/ao/quantization/quantizer/xnnpack_quantizer.py

```diff
@@ -114,7 +115,7 @@ def get_symmetric_quantization_config(
         if is_dynamic:
             act_observer_or_fake_quant_ctr = PlaceholderObserver  # type: ignore[assignment]
         else:
-            act_observer_or_fake_quant_ctr = HistogramObserver  # type: ignore[assignment]
+            act_observer_or_fake_quant_ctr = MinMaxObserver if ptq_act_use_minmax_observer else HistogramObserver  # type: ignore[assignment]
```

Comment:
Will all observers for mul then be MinMax? It seems like we are just using MinMaxObserver to get around the sys.floatmax, is there a way to use MinMaxObserver only for this case, and use HistogramObserver for the other activations?

to be fair, I don't truly understand how this will affect the numerics

cc. @kimishpatel

Reply:
we can but we'll need to implement https://github.com/pytorch/pytorch/blob/main/torch/ao/quantization/quantizer/xnnpack_quantizer.py#L314

this change does not change any existing tests btw, it's an optional flag that allows people to use minmax observer.

for mobilebert in executorch repo, do we have accuracy goals? or is it just to make sure that the quantization and lowering works?

Reply:
yeah as I commented in another comment. I dont think this is the right way to do it. Max would you mind fixing this yourself. If HistogramObserver cannot truly handle this and HistogramObsever does not seem the right place to fix this issue, then it belong to quantizer, who should make this decision. @mcr229 if you can do this, that would be great. I dont want this to be a global config.